src/compression/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+cmake_minimum_required(VERSION 3.9.0)
+
+project(realsense2-compression VERSION 1.0.0 LANGUAGES CXX C)
+# Save the command line compile commands in the build output
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
+
+set(DEPENDENCIES ${DEPENDENCIES} realsense2)
+
+file(GLOB COMPRESSION_SOURCES 
+ "*.hh"
+ "*.h"
+ "*.cpp"
+ "../ipDeviceCommon/*.h"
+ "../ipDeviceCommon/*.cpp"
+)
+
+add_library(${PROJECT_NAME} STATIC ${COMPRESSION_SOURCES})
+
+include_directories(${PROJECT_NAME}
+    ../../common
+    ../ipDeviceCommon
+)
+
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+
+add_dependencies(realsense2-compression 
+    lz4
+    libjpeg-turbo
+    zlib
```

Comment:
is zlib in use?

Reply:
It could be used instead of lz4. But default - it is not.